obs-browser-source.cpp

```diff
@@ -518,6 +520,10 @@ void BrowserSource::Update(obs_data_t *settings)
 		n_webpage_control_level = static_cast<ControlLevel>(
 			obs_data_get_int(settings, "webpage_control_level"));
 
+		n_javascript_active =
+			obs_data_get_bool(settings, "javascript_active");
+		n_javascript = obs_data_get_string(settings, "javascript");
```

Comment:
I would personally prefer if these were always in the same order as their declarations, here and throughout, as much as realistically possible.

Reply:
Alright, changed it, so it is now everywhere as "first javascript, then _active". (comes with the next commit as soon as I solve all the other topics listed below)